### PR TITLE
add a new bridge option, HOLO_ICE

### PR DESCRIPTION
**Pull request by trevj**
Tested with Simple SOCKS and uProxy (pull requests to follow).

[Review on Reviewable](https://reviewable.io/reviews/uproxy/uproxy-lib/209)

### CODE REVIEW

File: src/bridge/bridge.ts [hunk 1]
```diff
@@ -168,15 +181,21 @@ export class BridgingPeerConnection implements peerconnection.PeerConnection<
 
   private makeFromProviderType_ = (
       type:ProviderType) : peerconnection.PeerConnection<any> => {
-    if (type !== ProviderType.PLAIN && type !== ProviderType.CHURN) {
+    if (type !== ProviderType.PLAIN && type !== ProviderType.CHURN &&
```
Comment (bemasc) [inline src/bridge/bridge.ts]: Can this be a lookup in `ProviderType`?

File: src/bridge/bridge.ts [hunk 2]
```diff
@@ -190,11 +209,19 @@ export class BridgingPeerConnection implements peerconnection.PeerConnection<
...
   private makeChurn_ = (
       pc:freedom_RTCPeerConnection.RTCPeerConnection)
-      : peerconnection.PeerConnection<churn_types.ChurnSignallingMessage> => {
+      :peerconnection.PeerConnection<churn_types.ChurnSignallingMessage> => {
     log.debug('%1: constructing churn peerconnection', this.name_);
     return new churn.Connection(pc, this.name_);
   }
 
+  // Factored out for mocking purposes.
+  private makeHolographicIce_ = (
```
Comment (bemasc) [inline src/bridge/bridge.ts]: Maybe add "only"?

**Comment by bemasc** [comment]
:+1:

File: src/bridge/bridge.ts (hunk 2 above)
Comment (trevj) [inline src/bridge/bridge.ts]: Done.

File: src/bridge/bridge.ts (hunk 1 above)
Comment (trevj) [inline src/bridge/bridge.ts]: Done.

**Pull request merged**